di_website/spotlight/models.py

```diff
@@ -36,6 +50,24 @@ class Meta():
     ]
 
 
+class SpotlightLocationComparisonPage(HeroMixin, Page):
+    default_locations = StreamField([
+        ('locations', StructBlock([
+            ('name', TextBlock()),
+            ('geocode', TextBlock()),
+        ]))
+    ], null=True, blank=True, verbose_name='Default Locations')
+    content_panels = Page.content_panels + [
+        hero_panels(),
```

Comment:
Doesn't need a hero anymore since the design changed.

Reply:
Removed and updated